dockerfile.dev

```diff
@@ -29,5 +29,6 @@ EXPOSE 8000
 
 # Run migrations and start the development server on port 8000
 CMD python src/manage.py migrate && \ 
+    python src/manage.py collectstatic && \ 
```

Comment:
Please make the same change to the dockerfile.prod

Reply:
@goneall was going through the `dockerfile.prod` but couldn't find how we are starting the django server.
wanted to ask like do we manually run the commands while deploying or how is it?

Reply:
It is run under [supervisor](http://supervisord.org/).  The last line in the docker file starts it up.

[supervisor_api.conf](https://github.com/spdx/spdx-online-tools/blob/main/supervisor_api.conf) is the config file.

Supervisor starts up [gunicorn](https://gunicorn.org/) to run a [wsgi](https://wsgi.readthedocs.io/en/latest/what.html) script which run Django via wsgi.

Here's some background on how that is setup: https://docs.djangoproject.com/en/5.0/howto/deployment/wsgi/

Reply:
Thanks @goneall for this, I will make the appropriate changes now